integration-tests/jpa-oracle/src/test/java/io/quarkus/example/test/JPAFunctionalityInGraalITCase.java

```diff
@@ -0,0 +1,11 @@
+package io.quarkus.example.test;
+
+import io.quarkus.test.junit.SubstrateTest;
+
+/**
+ * Test various JPA operations running in SubstrateVM
+ */
+@SubstrateTest
```

Comment:
it has been renamed to @ NativeImageTest so please rebase your code and migrate to laster version